src/index.html

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+<head>
+    <meta charset="utf-8"/>
+    <!--    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />-->
```

Comment:
why did you comment it?

Reply:
Ya esta resuelto este tema del %PUBLIC_URL% 🚀